plugin/click.py

```diff
@@ -0,0 +1,19 @@
+import sublime
+import sublime_plugin
+
+
+class LspClick(sublime_plugin.TextCommand):
+    def __init__(self, view: sublime.View) -> None:
+        super().__init__(view)
+        self.click_count = 0
+
+    def run(self, edit: sublime.Edit, count: int, command: str, args: dict) -> None:
```

Comment:
Does it really make sense to have the flexibility of specifying `count`? For single clicks we don't need to use it and more than double click seems kinda not that useful? If we don't need that then we could call it `LspDoubleClickCommand` or even `LspOnDoubleClickCommand` since otherwise it sounds like it performs a double click itself.